src/client/components/JobCard/JobCard.styles.css

```diff
@@ -0,0 +1,18 @@
+.job-card-container {
+  background-color: transparent;
+  width: 507px;
+}
+
+.job-card-container h1 {
+  font-family: 'Roboto Mono', monospace;
+  font-size: 48px;
+  margin: 30px 0;
+}
+
+.job-card-container p {
+  font-family: 'Roboto Mono', monospace;
+  background-color: #f0f0f0;
+  height: 420px;
+  padding: 25px 20px;
+  font-size: 18px;
+}
```

Comment:
Everything looks super nice to me :) Have a small suggestion though. Can we rather add font-family in the container class instead of p and h1 to avoid redundancy?

Reply:
Since there is a Montserrat family-font as a global variable in the index.css I had to put the Roboto Mono in the h1 and p tag otherwise it would not work, but now that Daniel gived me permission to change the index.css I will make the acording changes :)